Makefile

```diff
@@ -0,0 +1,40 @@
+HOST = localhost:4000
+NPM_BIN = ./node_modules/.bin
+export PATH := $(NPM_BIN):$(PATH)
+
+# Install dependencies
+install:
+	@echo "Installing Pa11y..."
+	@npm install pa11y@^3
+	@echo "Installing GitHub Pages gem..."
+	@gem install github-pages
+
+# Build the site
+build:
```

Comment:
Does the makefile have a use outside of Circle? If not, are the `build` and `watch` tasks necessary?